app/common/auth/password-reset-confirm.controller.js

```diff
@@ -3,7 +3,7 @@ module.exports = PasswordResetConfirmController;
 PasswordResetConfirmController.$inject = ['$rootScope', 'PasswordReset', '$location', '$transition$'];
 function PasswordResetConfirmController($rootScope, PasswordReset, $location, $transition$) {
     var $scope = $rootScope.$new();
-    $scope.token = $transition$.params().token;
+    $scope.token = decodeURIComponent($transition$.params().token);
```

Comment:
I don't understand how this is working in production since it's the same there.
Are you sure ui-router doesn't decode the url component for us?